flink/src/main/java/org/apache/iceberg/flink/FlinkUtil.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.flink;
+
+import java.nio.ByteBuffer;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.apache.iceberg.expressions.BoundPredicate;
+import org.apache.iceberg.expressions.ExpressionVisitors;
+import org.apache.iceberg.expressions.Literal;
+import org.apache.iceberg.expressions.UnboundPredicate;
+
+public class FlinkUtil {
```

Comment:
It's good to move this class to `test` packages, I saw nobody from production files would call this.

Reply:
Sounds like it should be renamed as `FlinkFiltersUtil`

Reply:
I copyed the  code from the spark module, in spark, there are production files call this mothed.

In Flink, this class does only have methods related to filters. I think some methods related to flink util can be added to this class in the future. So I do not add the class in test packages,what do you think?

Reply:
The `DescribeExpressionVisitor`  is not actually format the flink filters to string,   instead it's formatting the iceberg's `Expression` to string.  So even if we want to introduce a flink utility ,  it's not a good idea to put the iceberg's visitor in this class.

Reply:
I'd prefer to abstract the common methods from flink and spark to a common utility.   How about introducing a `ExpressionsUtil` under package `org.apache.iceberg.expressions` ?  In that way, we could remove the `Spark3Util#describe` methods.